packages/facebook-oauth/package.js

```diff
@@ -0,0 +1,16 @@
+Package.describe({
+  summary: "Facebook OAuth flow",
+  version: "1.3.0"
+});
+
+Package.onUse(function(api) {
+  api.use('oauth2', ['client', 'server']);
+  api.use('oauth', ['client', 'server']);
+  api.use('http', ['server']);
+  api.use('underscore', 'server');
+  api.use('random', 'client');
+  api.use('service-configuration', ['client', 'server']);
```

Comment:
I don't think this package should depend on `service-configuration`, this should be in the config ui package.